wsclient/src/test/java/org/hpccsystems/ws/client/DFUFileDetailWrapperTest.java

```diff
@@ -332,6 +332,34 @@ public void testGetRecordMaxlength() throws Exception {
         assertNotNull(column);
     }
 
+    @Test
+    public void testSetDefault() throws Exception {
+        String setof = "RECORD\nSET OF UTF8 field3set {DEFAULT ([''])};\nEND;";
+        EclRecordWrapper info = DFUFileDetailWrapper.getRecordEcl(setof);
```

Comment:
add this to the constants at the top of this file. Also add a field before and after the SET in the record and ensure that they are still handled as expected

Reply:
updated